### ISSUE TITLE
Cannot set value of data object to a number

### ISSUE THREAD

**Issue by jeffblake** [vedmack/yadcf#346]
```
      data: [
        {value: 0, label: 'Valid'}
        {value: 1, label: 'Redeemed'}
        {value: 2, label: 'Out'}
        {value: 3, label: 'Revoked'}
        {value: 4, label: 'Transferred'}
        {value: 7, label: 'Unconfirmed'}
      ]
```

Breaks on line 2538:
`									options_tmp += "<option value=\"" + column_data[ii].value.replace(/"/g, '&quot;') + "\">" + column_data[ii].label + "</option>";
`

`VM32781:1 Uncaught TypeError: column_data[ii].value.replace is not a function(…)`

**Comment by vedmack** [comment]
Please provide a js playground test page with this issue

**Comment by vedmack** [comment]
fixed in 0.9.1.beta.3
